src/Symfony/Bridge/Twig/Tests/Extension/WorkflowExtensionTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Extension;
+
+use Symfony\Bridge\Twig\Extension\WorkflowExtension;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\Registry;
+use Symfony\Component\Workflow\Workflow;
+
+class WorkflowExtensionTest extends \PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (!class_exists('Symfony\Component\Workflow\Workflow')) {
+            $this->markTestSkipped('The Workflow component is needed to run tests for this extension.');
+        }
+    }
+
+    public function testHasPlace()
+    {
+        $subject = new \stdClass();
+
+        $marking = new Marking(array('ordered' => true, 'waiting_for_payment' => true));
+
+        $workflow = $this->getMock(Workflow::class, array(), array(), '', false);
```

Comment:
Anyway, I would use concrete implementation instead of mock.
I will be simpler.